pkg/sql/schemachanger/scbuild/internal/scbuildstmt/process.go

```diff
@@ -11,31 +11,58 @@
 package scbuildstmt
 
 import (
+	"reflect"
+
+	"github.com/cockroachdb/cockroach/pkg/settings"
 	"github.com/cockroachdb/cockroach/pkg/sql/schemachanger/scerrors"
 	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
 )
 
+var declarativeForSupportedAllOps = settings.RegisterBoolSetting(
+	"sql.schema_changer.declarative_for_all",
+	"when set to true the declarative schema changer is enabled for "+
+		"buth supported/unsupported operations which are full tested.",
```

Comment:
Several typos. This string is user-facing, please exercise more care when composing messages like these. I'm surprised your spell-checker didn't catch this.